plugins/woocommerce/src/Blocks/Shipping/PickupLocation.php

```diff
@@ -38,7 +38,7 @@ public function __construct() {
 	 */
 	public function init() {
 		$this->enabled          = $this->get_option( 'enabled' );
-		$this->title            = $this->get_option( 'title' );
+		$this->title            = $this->get_option( 'title', __( 'Pickup', 'woocommerce' ) );
```

Comment:
In some instances, title was missing on classic checkout. I assume its because the default is blank.